srcpkgs/gzdoom/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'gzdoom'
 pkgname=gzdoom
 version=4.2.4
-revision=1
+revision=2
 _tagdate=2019-07-09
 wrksrc="${pkgname}-g${version}"
 build_style=cmake
 configure_args="-DINSTALL_PK3_PATH=share/gzdoom"
 hostmakedepends="pkg-config"
-makedepends="SDL2-devel gtk+3-devel fluidsynth-devel libgme-devel libgomp-devel ppl-devel"
-depends="libfluidsynth gtk+3"
+makedepends="SDL2-devel gtk+3-devel fluidsynth-devel libgme-devel libgomp-devel ppl-devel libopenal-devel libsndfile-devel"
+depends="libfluidsynth gtk+3 libopenal libsndfile"
```

Comment:
Please break the makedepends line at 80 columns, and `libopenal` and `libsndfile` shouldn't need to be in depends, `xbps-src` should be able to detect them at build time.